custom_components/wellbeing/api.py

```diff
@@ -150,8 +151,8 @@ def _create_entities(data):
                 unit=PERCENTAGE
             ),
             ApplianceSensor(
-                name="CO2",
-                attr='CO2',
+                name="eCO2",
+                attr='ECO2',
                 unit=CONCENTRATION_PARTS_PER_MILLION,
                 device_class=SensorDeviceClass.CO2
```

Comment:
This sensor already exist in the A7 array. 

```
        a7_entities = [
            ApplianceSensor(
                name="eCO2",
                attr='ECO2',
                unit=CONCENTRATION_PARTS_PER_MILLION,
                device_class=SensorDeviceClass.CO2
            ),
```

You should move it to the common array instead of adding it again.

Reply:
changed it in the PR